somnio_starter/__brick__/{{project_name.snakeCase()}}/packages/user_repository/lib/src/user_repository.dart

```diff
@@ -18,7 +19,9 @@ class UserRepository {
   final ApiClient _apiClient;
   final AuthenticationClient _authenticationClient;
 
-  Future<User?> _onAuthStatusChanged(AuthenticationEvent event) async {
+  /// Maps the [AuthenticationEvent] to a [User] object.
+  @visibleForTesting
+  Future<User?> onAuthStatusChanged(AuthenticationEvent event) async {
```

Comment:
Can we keep this as private?

Reply:
We need it so we can test it and achieve higher coverage